housing_cooperative_base/wizard/rental_state_report_wizard.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import api, exceptions, fields, models
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class RentalStateReportWizard(models.TransientModel):
+    _name = "hc.rental.state.report.wizard"
+    _description = "Create Rental State Report"
+
+    name = fields.Char(compute="_compute_name", store=True)
+    building_ids = fields.Many2many(
+        comodel_name="hc.building", string="Buildings"
+    )
+    start = fields.Date(string="Start", required=True)
+    end = fields.Date(string="End", required=True)
+    lease_line_ids = fields.Many2many(
+        comodel_name="hc.lease.line", compute="_compute_lease_line_ids"
+    )
+
+    @api.multi
+    @api.depends("start", "end")
+    def _compute_name(self):
+        for wizard in self:
+            wizard.name = "rental_state_report_%s_%s" % (
+                wizard.start,
+                wizard.end,
+            )
+
+    @api.multi
+    @api.depends("building_ids", "start", "end")
+    def _compute_lease_line_ids(self):
+        self.ensure_one()
+
+        today = fields.Date.today()
+        lease_line_ids = self.env["hc.lease.line"].search([])
```

Comment:
Why are you using filtered on all lease lines and not on in the `search` function?

Also, shouldn't you filter on the state of lease lines?

Reply:
Somehow forgot you can add filter in `search()`, and yes I forgot > fixed!

Reply:
This will be fixed in the reports branch